tests/test1.in

```diff
@@ -33,12 +33,12 @@ end
 # Simple alias tests
 
 function foo
-    echo >foo.txt $argv
+    echo >/tmp/fish_foo.txt $argv
```

Comment:
Yes, that sounds good. Perhaps it's time to collect all the test_* directories in order to avoid cluttering the root directory?